api.go

```diff
@@ -123,11 +149,17 @@ func BuildDockerFile(p martini.Params, r render.Render, dockerInfo DockerInfo) {
 	err := GenerateDockerFile(dockerInfo, ".")
 
 	if err != nil {
-		r.JSON(500, map[string]interface{}{"success": false})
+		// this line can probably be removed but we need to figure what if anything we should return in responses
+		r.JSON(500, map[string]interface{}{"success": "false"})
+		makePutRequest(dockerInfo, "FAL")
+
 	}
 
 	logging.LogToFile("Dockerfile generated successfully...building container...")
 
 	go Build(buildLocation, dockerInfo.REPO_NAME, dockerInfo.PATH)
-	r.JSON(200, map[string]interface{}{"success": true})
+	// this line can probably be removed but we need to figure what if anything we should return in responses
+	r.JSON(200, map[string]interface{}{"success": "true"})
+	makePutRequest(dockerInfo, "BLD")
```

Comment:
This should probably be `SUC` as build is the default state api sets after it successfully calls builder the first time.